package.json

```diff
@@ -27,6 +27,7 @@
     "@plaiceholder/next": "^2.5.0",
     "@polkadot/api": "^10.10.1",
     "@polkadot/extension-dapp": "^0.44.6",
+    "@polkadot/keyring": "^12.6.2",
```

Comment:
Is this needed? I think we already inherit it from one of the other libraries

Reply:
are you referring to this: `import { decodeAddress } from "@polkadot/keyring";` ?

Reply:
Yeah this dependency

Reply:
Do you know why this would cause a build error now on Vercel? I'm getting `Cannot find module '@polkadot/keyring' or its corresponding type declarations.` 

When I run `yarn cache clean --all` and `yarn install` I do notice that the `keyring` package inside `node_modules` gets removed when on this branch

Reply:
Yeah that's strange I've just added it explicitly, should be ok for now